static/sass/areas/_permalink.scss

```diff
@@ -495,6 +495,37 @@
     margin-left: 8px;
 }
 
+/* adding tumblr */
+.permalink-sidebar .permalink-social .tumblr {
+    a {
+        background-color: #001935;
+        display: inline-block;
+        height: 20px;
+        border-radius: 3px;
+        text-decoration: none;
+        color: #fff;
+        font-family: Helvetica Neue,HelveticaNeue,Helvetica,Arial,sans-serif;
+        cursor: pointer;
+        box-sizing: border-box;
+    }
+
+    a .inner {
```

Comment:
I'd prefer the markup and presentation to be similar to the other sharing icons. So, icon-only, like Twitter/Facebook have. You should use an alt attribute on the "a" tag for "Post to Tumblr", for accessibility. And I'd make the SVG a static file like we use for Twitter/Facebook, and reference it with an img tag.

Reply:
I was trying to match the standard tumblr share button, but you're right, it would be better to match our existing icons. I'll try to extract the 't' part of the svg and just use a standard img element with an alt attribute. Thanks!